somefile.md

```diff
@@ -0,0 +1 @@
+somefile
```

Comment:
What is the reason of creating this file?

Reply:
This file is required, just in case
```suggestion
somefile2.md
```